### ISSUE TITLE
Adding scale_grad_by_freq option to torchrec embedding

### ISSUE THREAD

**Issue by fangleigit** [meta-pytorch/torchrec#1537]
In pytorch embedding module, it has scale_grad_by_freq option, to scale gradients by the inverse of frequency of the words in the mini-batch. I am thinking if this is also doable in torchrec, The motivation is that in RecSys or CTR prediction, the embedding table could be very sparse, thus the learning rate for those sparse entries should be larger.
